examples/model_selection/plot_validation_curve.py

```diff
@@ -49,3 +49,29 @@
                  color="navy", lw=lw)
 plt.legend(loc="best")
 plt.show()
+
+#####################################################################
+# The same plot can also be generated using a combination of GridSearchCV and
+# plotting module of scikit-learn.
+
+print(__doc__)
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+from sklearn.datasets import load_digits
+from sklearn.svm import SVC
+from sklearn.model_selection import GridSearchCV
+from sklearn.plot import plot_gridsearch_results
+
+digits = load_digits()
+X, y = digits.data, digits.target
+
+param_grid = {'gamma': np.logspace(-6, -1, 5)}
+gs = GridSearchCV(SVC(),
+                  param_grid=param_grid,
+                  cv=10, scoring="accuracy")
+
+gs.fit(X, y)
+plot_gridsearch_results(gs.cv_results_)
```

Comment:
You could use `fmt='%.1e'` to have better x-tick-labels in [the plot](https://12791-843222-gh.circle-artifacts.com/0/home/ubuntu/scikit-learn/doc/_build/html/stable/_images/sphx_glr_plot_validation_curve_002.png)

You could also use `plt.tight_layout()` here, to prevent x-labels to be cropped out of the figure.